service/training.rest.xml

```diff
@@ -0,0 +1,27 @@
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/rest-api-2.1.xsd" name="Product" displayName="Product REST API" version="2.0.0">
+    <resource name="Product" require-authentication="anonymous-all">
```

Comment:
As per naming conventions of REST APIs, you must define resource names in lower case letters. I would recommend you to please check the best practices and update the names.